settings.gradle

```diff
@@ -85,7 +85,8 @@ include 'static-analysis:pmd-rules'
  '1.9.2',
  '1.10.0',
  '1.11.0',
- '1.12.0'].each {
+ '1.12.0',
+ '1.13.0'].each {
```

Comment:
good catch on this.  release script should have been keeping this up to date, hmm.  and both 1.13.0 **and 1.13.1** should be listed here, as both are released